conftest.py

```diff
@@ -7,18 +7,29 @@
 import pytest
 
 from sybil import Sybil
-from sybil.parsers.rest import DocTestParser, PythonCodeBlockParser
+from sybil.parsers import myst, rest
 
 import svc_reg
 
 
-pytest_collect_file = Sybil(
+markdown_examples = Sybil(
     parsers=[
-        DocTestParser(optionflags=ELLIPSIS),
-        PythonCodeBlockParser(),
+        myst.DocTestDirectiveParser(optionflags=ELLIPSIS),
+        myst.PythonCodeBlockParser(),
```

Comment:
```suggestion
        myst.PythonCodeBlockParser(doctest_optionflags=ELLIPSIS),
```

The `PythonCodeBlockParser` also handles the doctests inside fenced code blocks.